src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+        public Event(String description) {
```

Comment:
Should there be an empty line before this constructor?
I notice that there are times you leave an empty line before constructors, and there are times you leave out the empty line.